velox/connectors/hive/HiveConfig.h

```diff
@@ -231,12 +231,6 @@ class HiveConfig {
   static constexpr const char* kS3UseProxyFromEnv =
       "hive.s3.use-proxy-from-env";
 
-  /// Timestamp unit for Parquet write through Arrow bridge.
```

Comment:
Not sure if we want to remove them here.  It's better to expose them as compile time constant somewhere for integration, testing, and documentation purpose.

Reply:
We discussed this offline; the problem here is that defining file-format specific options here creates a circular dependency. I started decoupling this because on Nimble some of the parameters cannot be serialized, so there would be a dependency from HiveConnector into the file format options (and the other way around).

Reply:
We can move these to Parquet directories, just need to somewhere so that we can refer to it when we set it up in integration, testing, and documentation